doc/quality_metrics/firing_rate.rst

```diff
@@ -37,26 +37,26 @@ With SpikeInterface:
 
 .. code-block:: python
 
-    import spikeinterface.toolkit as st
+    import spikeinterface.qualitymetrics as qm
 
     # Make recording, sorting and wvf_extractor object for your data.
     
-    firing_rate = st.compute_firing_rate(wvf_extractor)
+    firing_rate = qm.compute_firing_rate(wvf_extractor)
     # firing_rate is a dict containing the units' ID as keys,
     # and their firing rate across segments as values (in Hz).
 
 References
 ----------
 
-.. automodule:: spikeinterface.toolkit.qualitymetrics.misc_metrics
+.. automodule:: spikeinterface.qualitymetrics.misc_metrics
 
     .. autofunction:: compute_firing_rate
 
 
 Links to source code
 --------------------
 
-From `SpikeInterface <https://github.com/SpikeInterface/spikeinterface/blob/ae679aff788a6dd4d8e7783e1f72ec7e550c1bf9/spikeinterface/toolkit/qualitymetrics/misc_metrics.py#L52>`_
+From `SpikeInterface <https://github.com/SpikeInterface/spikeinterface/blob/master/spikeinterface/qualitymetrics/misc_metrics.py#L52>`_
```

Comment:
Yes I know. You are right.
I did that quickly but this permanent link would be broken anyway no ?
The plan was to make new links after this merge with the new file path.

Reply:
I thought the permanent link linked to the commit?
Thus it would point to an older version (which means the link might need to be updated), but will point to the correct line.

I don't know which one is best, just wanted to point out the issue ^^

Reply:
I think that once we release, we should point to the release version. Agreed?

Reply:
Sounds like a good idea!